<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalsWidget</class>
 <widget class="QDockWidget" name="GlobalsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Global Variables</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="AddressableItemList&lt;&gt;" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">CutterTreeView::item
{
    padding-top: 1px;
    padding-bottom: 1px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="indentation">
       <number>8</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ComboQuickFilterView" name="quickFilterView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionEditGlobal">
   <property name="text">
    <string>Edit Global Variable</string>
   </property>
   <property name="toolTip">
    <string>Edit Global Variable</string>
   </property>
  </action>
  <action name="actionDeleteGlobal">
   <property name="text">
    <string>Delete Global Variable</string>
   </property>
   <property name="toolTip">
    <string>Delete Global Variable</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboQuickFilterView</class>
   <extends>QWidget</extends>
   <header>widgets/ComboQuickFilterView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddressableItemList&lt;&gt;</class>
   <extends>QTreeView</extends>
   <header>widgets/AddressableItemList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
